content/altinn-studio/guides/payment/backend-manual/calculate-payment.en.md

```diff
@@ -14,6 +14,7 @@ The return value from the `CalculateOrderDetails` method indicates:
 - Currency
 - Order lines
 - Details of payment receiver. Used in receipt.
+- Details about the payer (optional), If you want to pre-fill this information with Nets Easy. Can be used in combination with Nets Easy's flag MerchantHandlesConsumerData, which we have exposed via appsettings.json NetsPaymentSettings.MerchantHandlesConsumerData. If it is set to true, payer details must be sent along, otherwise, it will fail.
```

Comment:
Gir det mening å utheve kode/filer sånn som dette? :)
```suggestion
- Details about the payer (optional), If you want to pre-fill this information with Nets Easy. Can be used in combination with Nets Easy's flag `MerchantHandlesConsumerData`, which we have exposed via `appsettings.json` `NetsPaymentSettings.MerchantHandlesConsumerData`. If it is set to true, payer details must be sent along, otherwise, it will fail.
```

Reply:
👍 Glipp